isis/src/control/objs/BundleSolutionInfo/unitTest.cpp

```diff
@@ -90,6 +90,8 @@ int main(int argc, char *argv[]) {
 
```

Comment:
please add history

Reply:
done

Reply:
and done for BundleSettings unitTest as well